todoapp/app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/data/TasksLocalDataSourceTest.kt

```diff
@@ -90,23 +80,13 @@ import org.mockito.Mockito.verify
             completeTask(newTask)
 
             // Then the task can be retrieved from the persistent repository and is complete
-            getTask(newTask.id, object : TasksDataSource.GetTaskCallback {
-                override fun onTaskLoaded(task: Task) {
-                    assertThat(task, `is`(newTask))
-                    assertThat(task.isCompleted, `is`(true))
-                }
-
-                override fun onDataNotAvailable() {
-                    fail("Callback error")
```

Comment:
Wanted to check if I'm understanding correctly that we are no longer testing the fail scenario?

Reply:
Yep, this `fail()` call was required for the test case to fail when the data is not available. We don't need it as we're not using callbacks.